NSSNamespace/Initial_Bangazon_Site
# Es attempt product type idfix

Database should now seed with six product subcategories (2 per general category).
Products added to the database should now include a ProductTypeId

To Test

git fetch
git checkout es-attempt-productTypeIdfix
delete your migrations folder
-delete your database
-reset the path for your database by entering export Bangazon_Db_Path=`"path/to/database.bangazon.db" (on a mac; for windows, use set instead of export)`
enter dotnet ef migrations add WhateverYouWantToCallYourMigration in your terminal
enter dotnet ef database update in your terminal
-reopen your database and double-check that customers, products, product types, and product type subcategories are seeded
-run through the app to ensure all features still work
